airflow/api_connexion/parameters.py

```diff
@@ -48,6 +45,27 @@ def format_datetime(value: str):
         )
 
 
+def check_limit(value: int):
+    """
+    This checks the limit passed to view and raises BadRequest if
+    limit exceed user configured value
+    """
+    max_val = int(conf.get("api", "maximum_page_limit"))
```

Comment:
```suggestion
    max_val = conf.getint("api", "maximum_page_limit")
```
See: https://github.com/apache/airflow/pull/9436